x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/TransactionFlyout/index.tsx

```diff
@@ -32,6 +38,61 @@ interface Props {
   waterfall: IWaterfall;
 }
 
+const ResponsiveFlyout = styled(EuiFlyout)`
```

Comment:
This feels like something that should be part of EUIFlyout

Reply:
++